### ISSUE TITLE
Showing generated images using Matplotlib, OpenCV, etc.

### ISSUE THREAD

**Issue by prp-e** [lucidrains/imagen-pytorch#23]
Greetings.
I successfully managed to generate a few images (from README file of the main repo) but don't know how I can make images in the `images` list to image files. It'd be helpful if you add a guide.

**Comment by giovannizinzi** [comment]
Hey prp,
Here's some code to display generated image using matplotlib (assuming your list is named 'images')
```
import torch
import torchvision.transforms as T
import matplotlib.pyplot as plt

image = images[0]
transformVector = T.ToPILImage()
image = transformVector(image).convert("RGB")
plt.imshow(image)
```

and for opencv, you could use: https://docs.opencv.org/3.1.0/dc/d2e/tutorial_py_image_display.html

If you see what looks like static (generated by the torch.randn lines), make sure you trained your model on a labeled dataset of corresponding .jpgs and .txts

**Comment by ricklon** [comment]
I added a line in the readme.md examples:
`torch.save(images, "images.pt" )`

Added the load command to the above example:
`images = torch.load("images.pt" )`

In each case, I just get a random noise image. What should I see?

**Comment by Netruk44** [comment]
@ricklon An uninitialized model will result in noise, as you're seeing. There is no pretrained model yet (as far as I'm aware, I may be wrong), you will have to wait or train your own.

**Comment by roryoday** [comment]
use 
`plt.imshow(torchvision.utils.make_grid(images, nrow=5).permute(1, 2, 0))`

**Comment by akshaytheau** [comment]
![image](https://user-images.githubusercontent.com/15026338/171987318-4305fb0e-a73f-4898-94c7-b7b4a3854af9.png)
 
 Can anyone please help me ,why I am getting this output and How can I improve it.

**Comment by alexanderalber** [comment]
from my limited understanding the readme-code is incomplete. you need some pictures with labels to train it on (instead of those random numbers) and you need to do more than one training-iteration.

**Comment by prp-e** [comment]
If anyone trained this, please let us know how.

**Comment by saiaman** [comment]
same here, images are not real ones..... trained model looks bot working
